pkg/config/config.go

```diff
@@ -802,7 +801,7 @@ func (conf *Config) updateFromCLI(c *cli.Context, baseFlags []cli.Flag) error {
 		conf.RTC.NodeIP = c.String("node-ip")
 	}
 	if c.IsSet("udp-port") {
-		conf.RTC.UDPPort = uint32(c.Int("udp-port"))
+		conf.RTC.UDPPort = rtcconfig.PortRange{Start: (c.Int("udp-port"))}
```

Comment:
nit: we can make it parse what's on the command line? it seems possible to be able to specify a range too? i.e. --udp-port "7882-7892"